dm/relay/meta_test.go

```diff
@@ -234,3 +235,65 @@ func (r *testMetaSuite) TestLocalMeta(c *C) {
 	currentDir := lm.Dir()
 	c.Assert(strings.HasSuffix(currentDir, cs.uuidWithSuffix), IsTrue)
 }
+
+func (r *testMetaSuite) TestLocalMetaPotentialDataRace(c *C) {
```

Comment:
could please use testify to rewrite this test? see https://github.com/pingcap/tiflow/issues/2164

Reply:
OK，I'll submit another PR to rewrite the test